packages/wdio-cli/src/constants.ts

```diff
@@ -509,19 +509,19 @@ export const QUESTIONNAIRE = [{
         const pattern = isBrowserRunner(answers) ? 'src/**/*.test' : 'test/specs/**/*'
         return getDefaultFiles(answers, pattern)
     },
-    when: /* istanbul ignore next */ (answers: Questionnair) => answers.generateTestFiles && /(mocha|jasmine)/.test(answers.framework)
+    when: /* istanbul ignore next */ (answers: Questionnair) => /(mocha|jasmine)/.test(answers.framework)
 }, {
     type: 'input',
     name: 'specs',
     message: 'What should be the location of your feature files?',
     default: (answers: Questionnair) => getDefaultFiles(answers, 'features/**/*.feature'),
-    when: /* istanbul ignore next */ (answers: Questionnair) => answers.generateTestFiles && answers.framework.includes('cucumber')
+    when: /* istanbul ignore next */ (answers: Questionnair) => answers.framework.includes('cucumber')
 }, {
     type: 'input',
     name: 'stepDefinitions',
     message: 'What should be the location of your step definitions?',
     default: (answers: Questionnair) => getDefaultFiles(answers, 'features/step-definitions/steps'),
-    when: /* istanbul ignore next */ (answers: Questionnair) => answers.generateTestFiles && answers.framework.includes('cucumber')
```

Comment:
Why do we remove `answers.generateTestFiles` here?

Reply:
Because this is what prevents the question being asked when you do not generate examples. Now we ask the user where they plan to place their specs/features so that we don't end up with a ToDo comment in their config file which has to be resolved later. Me me that looked messy